bio_hansel/main.py

```diff
@@ -260,20 +263,30 @@ def main():
 
     dfsummary = pd.DataFrame(subtype_results)
     dfsummary = dfsummary[SUBTYPE_SUMMARY_COLS]
+
     if output_summary_path:
-        dfsummary.to_csv(output_summary_path, sep='\t', index=None)
+        if args.json:
+            dfsummary.to_json("{}.json".format(output_summary_path))
```

Comment:
By default `to_json` uses `orient='columns'` which represents the data as a dict `{column -> {index -> value}}`. I think `orient='records'` (`[{column -> value},..., {column -> value}]`) might be easier to work with since the index is dropped from the output (I don't think it meant anything anyway).

Reply:
Yeah, I was trying to get around that. Didn't see that you can just use orient='records'. That'll make my processing on the IRIDA side easier.